packages/firebase_remote_config/firebase_remote_config/android/src/main/java/io/flutter/plugins/firebase/firebaseremoteconfig/FirebaseRemoteConfigPlugin.java

```diff
@@ -56,8 +58,7 @@ public void onDetachedFromEngine(FlutterPluginBinding binding) {
         cachedThreadPool,
         () -> {
           Map<String, Object> configProperties = getConfigProperties(remoteConfig);
-          Map<String, Object> configValues = new HashMap<>();
-          configValues.putAll(configProperties);
+          Map<String, Object> configValues = new HashMap<>(configProperties);
```

Comment:
Passing the initial values direct to the constructor. `putAll` unnecessary in this case.